src/Lumi/Components2/Banner.purs

```diff
@@ -0,0 +1,206 @@
+                                      }
+                                    )
+                                $ _ { content = [ title ]
+                                    }
+                              _ -> mempty
+                          , box
+                            $ S._align S.Center
+                            $ S._row
+                            $ S._flex
+                            $ _ { content =
+                                    [ case props.icon of
+                                        Just icon ->
+                                          box
+                                          $ S._alignSelf S.Start
+                                          $ S.onDesktop (S._alignSelf S.Center)
+                                          $ style_
+                                              ( css
+                                                  { marginRight: int 16
+                                                  }
+                                              )
+                                          $ _ { content = [ icon ]
+                                              }
+                                        _ -> mempty
+                                    ]
+                                    <> props.content
+                                }
+                          ]
+                      }
+                , Monoid.guard props.dismissable
+                    $ E.element R.button'
+                      { css: theme # toCSS dismissButtonStyle
+                      , children:
+                          [ Icon.icon
+                              { style: R.css { fontSize: "12px" }
+                              , type_: Icon.Remove
+                              }
+                          ]
+                      , onClick: capture_ $ setVisible \_ -> false
+                      , className: "banner-dismiss"
+                      }
+                ]
+            }
+  where
+    defaults :: Record BannerProps
+    defaults =
+      { component: Banner
+      , title: Nothing
+      , content: []
+      , dismissable: false
+      , icon: Nothing
+      }
+
+    dismissButtonStyle :: forall props. PropsModifier props
+    dismissButtonStyle =
+      S._interactive
+      <<< S._alignSelf S.Start
+      <<< style \(LumiTheme { colors }) ->
+            css
+              { ariaLabel: str "dismiss"
+              , border: str "none"
+              , background: str "none"
+              , color: color colors.black1
+              , padding: int 0
+              , "&:focus": nested $ css
+                  { boxShadow: str "none"
+                  , outline: str "none"
+                  }
+              }
+
+actionBanner :: Array JSX -> PropsModifier BannerProps
+actionBanner actions f =
+  f >>> \props -> props
+    { content =
+        [ box
+          $ S._column
+          $ S.onDesktop (S._row)
+          $ S._flex
+          $ S._align S.End
```

Comment:
So now the content still gets aligned to the end of this container. I think we don't want this `S._align S.end`:
![Screen Shot 2020-05-12 at 11 16 23](https://user-images.githubusercontent.com/2164548/81703830-77b08c00-9443-11ea-825b-95cf22231468.png)
![Screen Shot 2020-05-12 at 11 17 22](https://user-images.githubusercontent.com/2164548/81703837-78e1b900-9443-11ea-93e7-edb4fb0dfc4f.png)

Here's how I got it working:
```diff
  f >>> \props -> props
    { content =
         [ box
           $ S._column
-          $ S.onDesktop (S._row)
+          $ S._align S.Start
+          $ S.onDesktop (S._row <<< S._align S.Center)
           $ S._flex
-          $ S._align S.End
           $ _ { content =
                   [ box
                     $ S._flex
                     $ _ { content = props.content }
                   , box
                     $ S._row
                     $ S._align S.Center
-                    $ S._justify S.End
+                    $ S._alignSelf S.End
+                    $ S.onDesktop (S._alignSelf S.Center)
                     $ style_
```